apps/admin-gui/src/app/shared/components/dialogs/applications-list-columns-change-dialog/applications-list-columns-change-dialog.component.ts

```diff
@@ -0,0 +1,96 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { Attribute, AttributesManagerService } from '@perun-web-apps/perun/openapi';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { NotificatorService, PerunTranslateService } from '@perun-web-apps/perun/services';
+
+export interface ApplicationsListColumnsChangeDialogData {
+  columns: string[];
+  theme: string;
+  voId: number;
+  groupId?: number;
+}
+@Component({
+  selector: 'app-applications-list-columns-change-dialog',
+  templateUrl: './applications-list-columns-change-dialog.component.html',
+  styleUrls: ['./applications-list-columns-change-dialog.component.scss'],
+})
+export class ApplicationsListColumnsChangeDialogComponent implements OnInit {
```

Comment:
Please separate the logic and the UI. It is imho cleaner and again esier to expand and reuse. You can check the add ban dialogs for insipration.